app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,88 +4,112 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
     color : $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
```

Comment:
Ok, in that case it sounds like a legit intermediate solution. Bootstrap 3 has a `sr-only` class for elements that are only for screen readers: http://getbootstrap.com/css/#forms-inline